content/en/docs/components/notebooks/jupyter-tensorflow-examples.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Jupyter TensorFlow Examples"
+description = "Examples using Jupyter and TensorFlow in Kubeflow Notebooks"
+weight = 40
+
++++
+
+## Mnist Example
```

Comment:
It would be helpful to have a brief intro paragraph explaining what this paragraph is doing.